hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestRegionReplicaReplicationEndpoint.java

```diff
@@ -487,4 +488,25 @@ private void testRegionReplicaReplicationIgnores(boolean dropTable, boolean disa
       connection.close();
     }
   }
+  private void enableTableWithRetries(TableDescriptor htd, boolean createTableOperation) throws InterruptedException {
+    // Helper function to run create/enable table operations with a retry feature
+    boolean continueToRetry = true;
+    int tries = 0;
+    while (continueToRetry && tries < 100) {
+      try {
+        continueToRetry = false;
+        if (createTableOperation) {
+          HTU.getAdmin().createTable(htd);
+        } else {
+          HTU.getAdmin().enableTable(htd.getTableName());
+        }
+      } catch (IOException e) {
+        if (e.getCause() instanceof ReplicationException) {
+          continueToRetry = true;
+          tries++;
+          Thread.sleep(1000);
```

Comment:
Let's use `Uninterruptibles.sleepUninterruptibly(1, TimeUnit.SECONDS)` so as to avoid throws clause for `InterruptedException`. 

The import package: 
`import org.apache.hbase.thirdparty.com.google.common.util.concurrent.Uninterruptibles`